ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml

```diff
@@ -346,7 +346,7 @@ tests:
         make ci-deploy-for-subsystem
         oc get pods
         make enable-kube-api-for-subsystem
-        make test-kube-api
+        make subsystem-test-kube-api
```

Comment:
Should we also change the periodic test below this one too? https://github.com/openshift/release/pull/57226/files#diff-7daa53a9b53dff31b810aac9ce26dd96e5c1dc41abdcfded87a84b5801ff4469R385

Reply:
Yes I just wanted to see it passes first, not to delay the other PRs further. I will change it as well, thanks

Reply:
Done